README-CONAN.md

```diff
@@ -320,9 +320,51 @@ Do not use conan on the Cygwin Platform.  To build Exiv2 for Cygwin use CMake wi
 
 ### 2.4) MinGW Notes
 
-Team Exiv2 supports MinGW msys/2.  Team Exiv2 does not support MinGW msys/1.0.
+We support:
+  - on Windows: MinGW msys/2.
+  - on Linux: MinGW toolchains.
+ 
+We do not support:
+  - on Windows: MinGW msys/1.0.
 
-As with Cygwin, we do not recommend using conan to build on the MinGW/msys2 platform.  We recommend installing dependences (expat, zlib) with platform tools or build/install from source.
+Dependencies can be installed with platform tools or conan.
```

Comment:
with platform tools => "with the platform's default tools" or "with the platform's tools"